setup.py

```diff
@@ -42,6 +42,8 @@
         'pandas >= 0.17.1',
         'simplejson >= 3.8.1',
         'matplotlib >= 2.0.0',
+        "dataclasses >= 0.7; python_version <'3.7'",
+        'typing-extensions >= 3.7.4.1'
```

Comment:
`Literal` is part of Python 3.8. 
We should add `python_version < '3.8'`

Reply:
c.f. comment above.